pkg/sessionctx/variable/tidb_vars.go

```diff
@@ -1250,7 +1250,8 @@ const (
 	DefOptConcurrencyFactor                 = 3.0
 	DefOptForceInlineCTE                    = false
 	DefOptInSubqToJoinAndAgg                = true
-	DefOptPreferRangeScan                   = false
+	DefOptPreferRangeScan                   = true
+	DefOptPreferRangeThreshold              = 1
```

Comment:
This variable is not used, can we remove it?

Reply:
done